torch/_inductor/codegen/triton.py

```diff
@@ -660,6 +660,14 @@ def atan(x):
     def atanh(x):
         return f"tl.math.atanh({x})"
 
+    @staticmethod
+    def frexp0(x):
+        return f"tl.math.ilogb({x})"
+
+    @staticmethod
+    def frexp1(x):
+        return f"tl.math.ldexp({x}, -tl.math.ilogb({x}))"
```

Comment:
Why not make the operators `ilogb` and `ldexp`?

Reply:
Doing `ilogb` + `ldexp` to get `frexp` is a bit slower than just doing `frexp` and using `frexp` directly for the cpu device.

Reply:
Hrm looks like these functions aren't inlined in C++ but the compiler is able to cse the two frexp calls. Would be nicer if we could CSE at inductor level but I guess this is okay.

Reply:
Yes, I noticed that too. The compiler is however able to optimize the two calls into one.